packages/devtools_app/lib/src/memory/flutter/memory_chart.dart

```diff
@@ -919,7 +942,11 @@ class SelectedDataPoint extends LineChartMarker {
 
   // These are the alpha blended values.
   final List<Color> _androidColors = [
-    ColorUtils.WHITE, // Total dashed line (Total)
+    // TODO(terry): Dash crashes Canvas unimplemented in Flutter Web
+    //              use a green color on Web see Flutter issue/49882.
+    kIsWeb
+        ? const Color(0xff00ff00)
```

Comment:
again here, do we want this to be the same color as dartVmColors? Before the colors were distinct (WHITE vs GRAY), so I'd think we want them different here, too.